### ISSUE TITLE
When NVDA is on, user must press Esc twice to close showQuickPick dropdown

### ISSUE THREAD

**Issue by cait1inhennessy** [Microsoft/vscode#58000]
Repro Steps:
1. Show any dropdown with vscode.window.showQuickPick API
2. Turn on NVDA
3. Check whether user is able to close the dropdown by pressing Esc once

OR
1. Install Live Share extension for VSCode
2. Press customized keyboard shortcut for "Sign In using browser(Sign In)" to open the Sign In window in browser.
3. Navigate to the Name of the user logged in footer and press enter
4. Start NVDA 
5. Check whether user is able to close the Collaboration list on pressing Esc only once when NVDA is on

Actual Result:-
When NVDA is on user need to press Esc two times to close dropdown

Expected Result:-
User should be able to close the list items on pressing the dropdown only once. Command to perform any operation should be changed when NVDA is on.

User Impact:-
Screen Reader users find difficulty in using application if user needs to operate controls twice to perform an action.

MAS Reference:-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Comment by cleidigh** [comment]
@cait1inhennessy 
This is how a form element such as a drop-down (combobox in ARIA parlance) is supposed to do when using a compliant screenreader.  Below is a link to an issue from the authors of NVDA to describe it better than I can:

https://github.com/nvaccess/nvda/issues/4428

(probably should be marked 'as designed') but I will defer to others.

**Comment by cait1inhennessy** [comment]
@cleidigh Thanks a lot for the clarification.

**Comment by chrmarti** [comment]
Closing as-designed. Thanks!
